src/main/java/frc/robot/subsystems/Elevator.java

```diff
@@ -73,6 +76,8 @@ public Elevator() {
     currentLimit.SupplyCurrentLowerTime = 1;
     currentLimit.StatorCurrentLimitEnable = true;
     currentLimit.SupplyCurrentLimitEnable = true;
+    gear3 = new DutyCycleEncoder(MotorIDConstants.gear3ID, 1.0, ElevatorConstants.gear3Offset);
```

Comment:
There is a DutyCycleEncoderSim class you could use to test in sim FYI. Not required